docs/jwst/error_propagation/main.rst

```diff
@@ -0,0 +1,374 @@
+optional arrays of wavelength and relative response (as a function of wavelength).
+The combination of the scalar conversion factor and the 2-D response values is
+applied to the science data, including the SCI and ERR arrays, as well as the
+variance (VAR_POISSON, VAR_RNOISE, and VAR_FLAT) arrays. The correction values
+are divided into the SCI and ERR arrays, and the square of the correction values 
+are divided into the variance arrays.  For details of the photom correction, see
+`Photom
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/photom/index.html>`_
+
+
+Cube build
+~~~~~~~~~~
+In the output spectral cube, the ERR extension contains the uncertainty on the 
+SCI values.  For details of the cube build step, see `CubeBuild
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/cube_build/index.html>`_
+
+
+Stage 2 WFS&C Processing pipeline - calwebb_wfs-image2
+++++++++++++++++++++++++++++++++++++++++++++++++++++++
+This pipeline processes Wavefront Sensing and Control (WFS&C) images, which
+duplicates the processing applied to regular science imaging, with the exception
+of image resampling. The steps in this pipeline are background image subtraction, 
+assign_wcs, flat_field, photom, and resample.  Of these steps, below are the steps 
+that alter the ERR or VAR arrays of the science ramp data and output those altered
+arrays in the science product that is passed to the next step in the pipeline.
+For the steps not listed below, those arrays do not exist or are propagated
+unchanged to the next pipeline step.
+
+
+Flat_field
+~~~~~~~~~~
+{NOTE get from the latest online readthedocs for the step once James updates it.}
+The SCI array from the flat-field reference file is divided into the VAR_POISSON
+and VAR_RNOISE arrays of the science data set. The variance in the flat is
+calculated in this step; the ERR array is then recalculated as these three
+variances added in quadrature.  The newly created VAR_FLAT array and the updated
+VAR_POISSON and VAR_RNOISE are output to the science products(s).  For details of
+the flat fielding correction, see `Flatfielding
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/flatfield/index.html>`_
+
+
+Photom
+~~~~~~ 
+The calibration information includes a scalar flux conversion constant, as well as
+optional arrays of wavelength and relative response (as a function of wavelength).
+The combination of the scalar conversion factor and the 2-D response values is
+applied to the science data, including the SCI and ERR arrays, as well as the
+variance (VAR_POISSON, VAR_RNOISE, and VAR_FLAT) arrays. The correction values
+are divided into the SCI and ERR arrays, and the square of the correction values 
+are divided into the variance arrays.  For details of the photom correction, see 
+`Photom
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/photom/index.html>`_
+
+
+Stage 3 pipelines
+-----------------
+Stage 3 pipelines perform corrections that work with multiple exposures and in
```

Comment:
They aren't all "corrections", so change to something a bit more generic, like "operations"